src/renderer/StoreChangeObserver.tsx

```diff
@@ -188,6 +190,21 @@ const StoreChangeObserver = () => {
     ipc.setConfidenceLinesEnabled(words !== null);
   }, [isShowingConfidenceUnderlines, words]);
 
+  // Autosave after currentProject changes.
+  useEffect(() => {
+    if (
+      currentProject &&
+      currentProject.transcription &&
+      currentProject.projectFilePath
+    ) {
+      if (collab !== null && collab.isHost) return; // Disallow autosave if you are not the host in collab
```

Comment:
`!collab.isHost`, the way you've written it, it will early exit if they _are_ the host